CMakeLists.txt

```diff
@@ -42,10 +43,10 @@ SET(PROJECT_USE_CMAKE_EXPORT True)
 
 # Create different building options
 OPTION(ENABLE_VECTORIZATION "Enable vectorization and further processor-related optimizations" OFF)
-OPTION(BUILD_PYTHON_INTERFACE "Build the python binding" ON)
-OPTION(BUILD_UNIT_TESTS "Build the unitary tests" ON)
+OPTION(BUILD_PYTHON_INTERFACE "Build the python binding" OFF)
```

Comment:
Do not change the default compilation options.
You can modify for yourself using `make edit_cache`